src/sentry/static/sentry/app/components/asyncComponent.tsx

```diff
@@ -144,8 +162,11 @@ export default class AsyncComponent extends React.Component {
     document.removeEventListener('visibilitychange', this.visibilityReloader);
   }
 
+  api: any;
```

Comment:
Should we have an interface for the ApiClient?

Reply:
I tried to type the `Client` object during Hack Week, but I wasn't able to get too far. 

A `.d.ts` file may be provided as an alternative.

Reply:
Added this to the discussion items for the next TSC. I agree getting type definitions for this class should be a priority.